CMakeLists.txt

```diff
@@ -98,6 +98,18 @@ option(SSG_SCIENTIFIC_LINUX_DERIVATIVES_ENABLED "If enabled, Scientific Linux de
 
 option(FORCE_VALIDATE_EVERYTHING "If enabled, perform all validation tests regardless of oscap version requirements. By default, the build system may not perform validation of some content types if the openscap used to build it is not up-to-date" FALSE)
 
+# Remove this option when we would like to run ansible-lint and yamllint against our playbooks by default.
+# Right now these checks are not performed and need to be enabled by adding -DANSIBLE_CHECKS=ON to cmake
+# before running ctest.
+option(ANSIBLE_CHECKS "Set to ON to enable ansible-lint and yamllint checks" ON)
+if(ANSIBLE_CHECKS)
+    # ANSIBLE_CHECKS needs to have individual playbooks per rule enabled
+    if(NOT SSG_ANSIBLE_PLAYBOOKS_PER_RULE_ENABLED)
+        message(WARNING "ANSIBLE_CHECKS is set to ON but it requires SSG_ANSIBLE_PLAYBOOKS_PER_RULE_ENABLED. Enforcing its value to ON.")
+        set(SSG_ANSIBLE_PLAYBOOKS_PER_RULE_ENABLED ON)
```

Comment:
Is there a special need relationship between `ANSIBLE_CHECKS` and `ANSIBLE_PLAYBOOKS_PER_RULE_ENABLED` that requires it build the rule playbooks?
Forcing build and installation of per rule playbooks por the purpose of running linters seems awkward to me.

Reply:
I think the issue is that the testing scrip `tests/ansible_playbook_check.sh` doesn't care about which playbooks it is checking. It just searches the build directory.